routes/api/hacker.js

```diff
@@ -905,6 +905,23 @@ module.exports = {
             Controllers.Hacker.sentDayOfEmail
         );
 
+        hackerRouter.route("/discord").post(
+          Middleware.Auth.ensureAuthenticated(),
+          Middleware.Auth.ensureAuthorized(),
```

Comment:
Don't you need to also give hackers the authorization to access this new route?

Reply:
> Don't you need to also give hackers the authorization to access this new route?

This route won't be used directly by Hackers. We will have a discord bot that has an admin account that is signed-in. Hackers will enter their email on discord private messages with the bot and the bot will forward the request to this endpoint.

Reply:
This isn't the most ideal solution but considering the timeframe that we are on, we don't have much of a choice.

Reply:
Since I don't think Discord will be going away as a platform that is used at Hackathons over the next couple years (just because of ease of use). I'll write a version 2.0 of the bot and this endpoint in the refactor that uses a much more technically sound solution. Where the user's receive e-mails when they are accepted with confirmation tokens attached, but once again due to time-frame constraints we were unable to do that this year.

Reply:
SG. So admins should have access to this endpoint. If I recall correctly, we still need to update the file with all of the routes so that admins have access to this endpoint

Reply:
> SG. So admins should have access to this endpoint. If I recall correctly, we still need to update the file with all of the routes so that admins have access to this endpoint

Yep! Good catch.